app/assets/xml/search.xml.erb

```diff
@@ -0,0 +1,9 @@
+<% helper = Module.new { extend ApplicationHelper } %>
+<?xml version="1.0" encoding="UTF-8"?>
+ <OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/">
+   <ShortName><%= helper.community_name %> Search</ShortName>
+   <Description>Find posts from the <%= helper.community_qualified_name %>.</Description>
```

Comment:
We previously had a `<Tags>` attribute with the value "software development" here. I removed this for now, since it's optional and the current one wouldn't work across communities. I was thinking about using `SiteConfig.suggested_tags`, but not sure if that's really what we want.